From: Sean Whitton <spwhitton@spwhitton.name>
Date: Fri, 26 Feb 2016 09:19:11 -0700
Subject: fix autoloads test

dash-el is currently outside of the dh_elpa infrastructure and so has no
autoloads file.  So hack the test to use another.
---
 test/flycheck-test.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/flycheck-test.el b/test/flycheck-test.el
index a35a82a..1ca80ab 100644
--- a/test/flycheck-test.el
+++ b/test/flycheck-test.el
@@ -635,7 +635,7 @@ and extension, as in `file-name-base'."
 
 (ert-deftest flycheck-autoloads-file-p/autoloads-with-backing-file ()
   :tags '(utility)
-  (flycheck-ert-with-file-buffer (locate-library "dash-autoloads")
+  (flycheck-ert-with-file-buffer (locate-library "shut-up-autoloads")
     (should (flycheck-autoloads-file-p))))
 
 (ert-deftest flycheck-autoloads-file-p/a-plain-file ()
